x-pack/solutions/security/plugins/security_solution/public/flyout/attack_details/hooks/use_find_attack_details_by_id/index.ts

```diff
@@ -0,0 +1,165 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { HttpSetup, IHttpFetchError, ResponseErrorBody } from '@kbn/core/public';
+import type {
+  AttackDiscoveryFindResponse,
+  AttackDiscoveryFindInternalResponse,
+} from '@kbn/elastic-assistant-common';
+import {
+  API_VERSIONS,
+  ATTACK_DISCOVERY_FIND,
+  ATTACK_DISCOVERY_INTERNAL_FIND,
+  transformAttackDiscoveryAlertFromApi,
+} from '@kbn/elastic-assistant-common';
+import type { QueryObserverResult, RefetchOptions, RefetchQueryFilters } from '@kbn/react-query';
+import { useQuery } from '@kbn/react-query';
+import { useCallback, useRef } from 'react';
+
+import { useAppToasts } from '../../../../common/hooks/use_app_toasts';
+import * as i18n from './translations';
+import { useKibanaFeatureFlags } from '../../../../attack_discovery/pages/use_kibana_feature_flags';
+
+type ServerError = IHttpFetchError<ResponseErrorBody>;
+
+interface Props {
+  id: string;
+  http: HttpSetup;
+  isAssistantEnabled: boolean;
+  refetchOnWindowFocus?: boolean;
+}
+
+type AttackDiscoveryInternalItem =
+  | AttackDiscoveryFindInternalResponse
+  | (AttackDiscoveryFindResponse extends { data: (infer U)[] }
+      ? ReturnType<typeof transformAttackDiscoveryAlertFromApi>
+      : unknown);
+
+export type AttackDiscoveryItem = AttackDiscoveryFindResponse extends { data: (infer U)[] }
+  ? ReturnType<typeof transformAttackDiscoveryAlertFromApi>
+  : unknown;
+
+interface UseFindAttackDiscoveryById {
```

Comment:
In the `flyout` folder we try to have a short description for each property in all interface we create. Can you update this one and the one above accordingly to stay consistent?